src/plugins/indicator_libmessaging.py

```diff
@@ -1,14 +1,18 @@
+"""
+src/plugins/indicator_libmessaging.py
+=====================================
+"""
 # -*- coding: utf-8 -*-
-
 import gi
-gi.require_version('MessagingMenu', '1.0')  # noqa:E402
 from gi.repository import MessagingMenu
-
```

Comment:
PEP8 says:
`You should put a blank line between each group of imports.`
https://www.python.org/dev/peps/pep-0008/#imports

Reply:
Okay, from next time i'll try to put a blank line between each group of imports, and will update the code as soon as possible.